src/node_api.cc

```diff
@@ -113,22 +113,22 @@ static v8::Local<v8::Function> V8LocalFunctionFromJsValue(napi_value v) {
 
 // Wrapper around v8::Persistent that implements reference counting.
 class Reference {
- public:
+ protected:
```

Comment:
I don't feel strongly about it. Was reasoning that subclassing ```Reference``` might happen at some point. Will change to ```private```.